src/test/kotlin/org/rust/ide/annotator/RsErrorAnnotatorTest.kt

```diff
@@ -3107,6 +3107,18 @@ class RsErrorAnnotatorTest : RsAnnotatorTestBase(RsErrorAnnotator::class) {
         const _: i32 = 1;
     """)
 
+    fun `test no E0603 for module with multiple declarations`() = checkDontTouchAstInOtherFiles("""
+        //- main.rs
```

Comment:
nit: Looks like code should be shifted left to have the same style as other tests